build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
   id 'com.github.ben-manes.versions' version '0.36.0'
   id 'com.github.hierynomus.license' version '0.16.1'
   id 'io.spring.dependency-management' version '1.0.11.RELEASE'
-  id 'net.ltgt.errorprone' version '1.3.0'
+  id 'net.ltgt.errorprone' version '2.0.0'
```

Comment:
Any reason to update this package?

Reply:
yes, because if you have a later version of the JDK installed (like I had 16) then the compile would fail.